jobs/probe_scraper.sh

```diff
@@ -20,17 +19,8 @@ mkdir $CACHE_DIR $OUTPUT_DIR
 # Finally run the scraper.
 python probe_scraper/runner.py --outdir $OUTPUT_DIR --tempdir $CACHE_DIR
 
-# Recursively GZIP the files in the output dir. Each file
-# will be compressed separately, creating a 'file.gz' file.
-gzip -9 -r $OUTPUT_DIR
-
-# Drop the '.gz' extension from the files.
-find $OUTPUT_DIR -type f -name '*.gz' | while read f; do mv "$f" "${f%.gz}"; done
-
 # Upload to S3.
```

Comment:
nit: do you think it's worth mentioning here that gzip'ing will be handled by CloudFront? You know, in case we forget in the future.

Reply:
Done. I've added a comment.